docs/pages/repo/docs/core-concepts/monorepos/code-generation.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Code Generation
+description: Extend your Turborepo with new apps, and packages. Create new empty workspaces, copy existing workspaces, add workspaces from remote sources (just like `create-turbo`!) or run custom generators defined using Plop configurations.
+---
+
+import Callout from "../../../../../components/Callout";
+
+# Code Generation
+
+Splitting your monorepo into individual workspaces is a great way to organize your code, speed up tasks, and improve the local development
+experience. One of the benefits of this approach is that it's easy to generate new source code for packages, modules,
```

Comment:
This sentence reads kind of weird. Using a monorepo doesn't make it easy to generate new source code. The tool we're introducing does.

Reply:
I agree, this was originally how I had positioned it but this update was suggested from @anthonyshew - but I might have missed the mark on it a bit. 

@anthonyshew do you want to tweak this?